nucleus/grizzly/config/src/main/java/org/glassfish/grizzly/config/ssl/JSSE14Support.java

```diff
@@ -20,61 +21,42 @@
 import java.io.ByteArrayInputStream;
 import java.io.IOException;
 import java.io.InputStream;
+import java.lang.System.Logger;
 import java.net.SocketException;
 import java.security.cert.Certificate;
 import java.security.cert.CertificateFactory;
 import java.security.cert.X509Certificate;
-import java.util.logging.Level;
-import java.util.logging.Logger;
+
 import javax.net.ssl.HandshakeCompletedEvent;
 import javax.net.ssl.HandshakeCompletedListener;
 import javax.net.ssl.SSLEngine;
 import javax.net.ssl.SSLException;
 import javax.net.ssl.SSLSession;
 import javax.net.ssl.SSLSocket;
 
-import org.glassfish.grizzly.Grizzly;
-// START SJSAS 6439313
-// END SJSAS 6439313
-
-/* JSSESupport
-
-   Concrete implementation class for JSSE
-   Support classes.
+import static java.lang.System.Logger.Level.DEBUG;
+import static java.lang.System.Logger.Level.ERROR;
 
-   This will only work with JDK 1.2 and up since it
-   depends on JDK 1.2's certificate support
-
-   @author EKR
-   @author Craig R. McClanahan
-   Parts cribbed from JSSECertCompat
-   Parts cribbed from CertificatesValve
-*/
 
```

Comment:
Should we remove this comment?

Reply:
The comment was copy pasted from JSSESupport which came from Tomcat - which now has completely different sources. In our JSSESupport is this comment too. Those two mentioned classes, JSSECertCompat and CertificatesValve are not on classpath and - probably they existed around year 2000 somewhere (aha, CertificatesValve is in tomcat4, 2001, I did not find the second one), but without package names it doesn't even make sense to have them here.
I think the JSSE14Support and JSSESupport could be merged to one, probably in next PR. We don't support Java 1.2 and 1.4 any more ...